tests/integration/pilot/common/routing.go

```diff
@@ -1,4 +1,6 @@
+//go:build integ
 // +build integ
```

Comment:
why this change?

Reply:
"smart"  ide auto inserting it 🙂will revert